### PR TITLE
Remove comments 1625

### PR DESCRIPTION
**Pull request by m4r10k**
### 1. What does this pull request do?
Remove comments which are useless due to code changes.

### CODE REVIEW

**Comment by miekg** [comment]
Note 100% removing the comments will make it right; I still think we can leave the loop normally and have something slightly too large; I double check after the 1.1.1 release.

**Comment by m4r10k** [comment]
OK, I understand +1

**Comment by m4r10k** [comment]
You have more overview about what exactly  is going on there in which case!
But if you need something to test, let me know then!

File: request/request.go [hunk 1]
```diff
@@ -217,6 +217,9 @@ func (r *Request) Scrub(reply *dns.Msg) (*dns.Msg, Result) {
 			re = m - 1
 			continue
 		}
+		if rl == size {
+			break
+		}
 	}
 	// We may come out of this loop with one rotation too many as we don't break on rl == size.
```
Comment (chrisohaver) [inline request/request.go]: This one too, if I'm not mistaken.

**Comment by chrisohaver** [comment]
lgtm

**Comment by miekg** [comment]
Full fix is in https://github.com/coredns/coredns/pull/1631
Tried to make a test case for the new `break`, but ended up doing more as we also need to take that OPT record into account (get re-added in SizeAndDo)

**Pull request closed without merging**